### PR TITLE
Ability to provide File or File Path for the Domain file manually

### PR DESCRIPTION
**Pull request by RogueSMG**
Changed the static 'alive.txt' path to be able to pass any file or file path directly as an argument from the CLI.

### CODE REVIEW

**Comment by dark-warlord14** [comment]
I made some changes. I would really appreciate if you clone it and do a fresh install and check if everything is fine or not. Then we will merge this. Thank you for the efforts you put in to make the script more cool.

Regards,
@dark-warlord14

**Comment by RogueSMG** [comment]
Hey @dark-warlord14, I reviewed the changes and updated some minor issues and mods. For now, I guess this is pretty much fine. Nonetheless, do have a look. And happy to help :smile:

File: script.sh [hunk 1]
```diff
@@ -48,7 +47,7 @@ mkdir db
 
 linkf=~/tools/LinkFinder/linkfinder.py
 
-for i in $(cat $cwd/$1)
+for i in $(cat $1)
```
Comment (RogueSMG) [inline script.sh:48]: Using pwd doesn't work if an absolute path to the file is given. This happens as it tries to replace the whole path twice, both for the $cwd and $1 which ends up getting appended like /current/path//path/to/file. Thus Removed it and printed the location of the results file insted once the script execution is finished.

**Comment by RogueSMG** [comment]
Hey, I resolved the minor errors in the script. Also updated the install script to add an alias on the fly. It's probably good to go now! Have a look though, and let me know if anything still messes up :wink:

**Comment by dark-warlord14** [comment]
It looks good. Let's get this into the master..
Can you please add the steps to source the alias and how it works in the Readme file. It's better if you write it from your perspective.

Best regards,
@dark-warlord14

**Pull request merged**